Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 name = "starship"
 version = "1.2.1"
-edition = "2021"
 authors = ["Starship Contributors"]
-homepage = "https://starship.rs"
+build = "build.rs"
```

Comment:
It seems that the plugin wants to reorg our TOML files to be alphabetized within each table. Are we okay with that?

Reply:
Looks like it's a deliberate Cargo.toml sorting order rather than alphabetical:
https://github.com/dprint/dprint-plugin-toml/blob/main/src/cargo.rs#L17

I personally like the sound of that 👍